website/source/docs/faq.html.md

```diff
@@ -26,3 +26,27 @@ a [gossip protocol](/docs/internals/gossip.html). The consensus protocol is stro
 consistent, and is used for all state replication and scheduling. The gossip protocol
 is used to manage the addresses of servers for automatic clustering and multi-region
 federation. This means all data that is managed by Nomad is strongly consistent.
+
+## Q: Is Nomad's `datacenter` parameter the same as Consul's?
+
+No. Unfortunately Nomad and Consul take slightly different approaches to
```

Comment:
No. For those familiar with Consul, Consul's notion of a datacenter is more equivalent to a Nomad's region. Nomad supports grouping nodes into multiple datacenters, which should reflect nodes being colocated, while being managed by a single set of Nomad servers. 

Consul on the other hand does not have this two tier approach to servers and agents and instead relies on federation to create larger logical clusters. 

<Your third paragraph>